cmd/root.go

```diff
@@ -76,39 +79,65 @@ func New(o *options.Options) *cobra.Command {
 	return cmd
 }
 
-// rootCmd runs scorecard checks given a set of arguments.
-func rootCmd(o *options.Options) error {
-	var err error
-	var repoResult scorecard.Result
+// Build the list of repositories to scan, honoring --repos > --org > --local > --repo/pkg-managers.
+func buildRepoURLs(ctx context.Context, o *options.Options) ([]string, error) {
+	// --repos has highest precedence
```

Comment:
FYI: since only one option can be set at a time, precedence doesn't really matter here.

Reply:
comment changed